WEB/Src/PerformanceCollector/PerformanceCollector/Perf.csproj

```diff
@@ -28,7 +28,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-    <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="1.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="5.0.0" />
```

Comment:
we cannot do this without confirming there were no breaking changes between 1 to 5. Else apps using this library may be forced to 5.0.0 version.
We can do the minimum version - like 2.1.X should be fine.